app/components/content-header.hbs

```diff
@@ -1,7 +1,11 @@
-<AuContentHeader @titlePartOne="Vlaanderen" @titlePartTwo="is linked data">
-  <img
-    sizes="50vw"
-    src={{"/assets/loket-header-1600.jpg"}}
-    alt="Foto van een laptop met daarop het vlaanderen logo."
-  />
-</AuContentHeader>
+<div style="height: 15rem; min-height: 15rem;">
```

Comment:
@snenenenenenene **suggestion (non-blocking)** Move inline css in `/styles` folder. 

Inline CSS is bad practice and should only be used as a last resort. I open a followup ticket to plan CSS cleaning: https://binnenland.atlassian.net/browse/BNB-171